### PR TITLE
Fix an issue in Federation when a type definition is not found for a type extension

### PR DESCRIPTION
**Pull request by kamilkisiela**
[Failing test](https://github.com/kamilkisiela/graphql-hive/actions/runs/4205307339/jobs/7297201194)
Apollo Federation struggles with `extend type EntityWithoutDefinition`, this PR fixes it

### CODE REVIEW

File: scripts/patch-manifests.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /*
   https://github.com/octokit/webhooks-methods.js/issues/45
-
```
Comment (dotansimha) [inline scripts/patch-manifests.js:3]: any reason for all of those extra spaces? 😆

File: patches/@octokit__webhooks-methods@3.0.1.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+index 7c6e9a1af624be81938679edba36a046fd78f5f9..3dd36bc7573673a082dd8cc4910e0e3aca51c4a9 100644
+--- a/package.json
++++ b/package.json
+@@ -42,6 +42,5 @@
+   },
+   "source": "dist-src/index.js",
+   "types": "dist-types/index.d.ts",
+-  "main": "dist-node/index.js",
+-  "module": "dist-web/index.js"
++  "main": "dist-node/index.js"
```
Comment (dotansimha) [inline patches/@octokit__webhooks-methods@3.0.1.patch:11]: this one is something else? or it's here on purpose?

File: scripts/patch-manifests.js (hunk 1 above)
Comment (kamilkisiela) [reply scripts/patch-manifests.js:3]: `prettier --write`

File: patches/@octokit__webhooks-methods@3.0.1.patch (hunk 2 above)
Comment (kamilkisiela) [reply patches/@octokit__webhooks-methods@3.0.1.patch:11]: We have two versions

Comment (kamilkisiela) [reply patches/@octokit__webhooks-methods@3.0.1.patch:11]: indirect dependencies

**Pull request merged**